templates/manage/users_list.tpl.html

```diff
@@ -133,11 +133,11 @@
         </td>
         <td width="20%">
           <a href="{$core.rel_url}manage/users.php?cat=edit&id={$list[i].usr_id}"
-                   title="{t}update this entry{/t}">{$list[i].usr_full_name|default:$list[i].usr_email}</a>
+                   title="{t}update this entry{/t}">{$list[i].usr_full_name|default:$list[i].usr_email|escape:"html"}</a>
```

Comment:
Emails are secured on input. Is this really an issue?

Reply:
Probably not an issue, but I would be consistent and escape anyway.

puts less effort just escape, than explaining why this is omitted each time somebody tries to review templates ;)

Reply:
OK

Reply:
It display fullName

<img width="294" alt="image" src="https://user-images.githubusercontent.com/31820707/141292722-6a30d2b8-2b44-455f-b0e2-64198f1f0808.png">